wgsl/index.bs

```diff
@@ -10133,78 +9817,18 @@ The following are reserved words:
 
```

Comment:
Is `unorm` a type?

Reply:
unorm is not a type.  When it's used it's as part of a context-dependent name for a channel in a texel format type.

e.g. rgba8unorm

Reply:
Do we ever have it on its own?

Reply:
No